astropy/coordinates/sky_coordinate.py

```diff
@@ -833,6 +833,13 @@ def to_string(self, style='decimal', **kwargs):
         lonargs.update(kwargs)
         latargs.update(kwargs)
 
+        if style == 'hmsdms' :  # for hmsdms, want to adjust precision of RA to be one more than 
+                                # that or dec, because hours are 15 * cos(dec) bigger than degrees.
+            # default case; precision = 0 for both
+            if not latargs.has_key('precision') : lonargs['precision'] = 1
+            else :
+               lonargs['precision'] = latargs['precision'] + 1
```

Comment:
It looks to me like there's three spaces instead of four at the last indent of that line?  PEP8 recommends 4-spaces always, and we try to do that uniformly in astropy to make sure we don't start running into indent-conflicts (that can lead to very difficult-to-debug problems...)